docs/tutorials/flowsom.ipynb

```diff
@@ -0,0 +1,346 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95ff14ef-3a04-406e-b3a6-fd411d2fb197",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adata.var"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0a35d260-95a1-4f6d-a31d-47836ad6718d",
+   "metadata": {},
+   "source": [
+    "As you can see, the dataset consists of numerous channels and some meta-variables such as individual and label."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5d5013f1-44ad-4072-b835-0850054b46ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adata[:, \"label\"].to_df()[\"label\"].value_counts(dropna=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a40b930c-6605-4f1c-b408-713dd74118ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adata[:, \"individual\"].to_df()[\"individual\"].value_counts(dropna=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7d4b0412-bc40-4c5e-81fc-d73b8ed95510",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "channels = [\n",
+    "    \"DNA1\",\n",
+    "    \"DNA2\",\n",
+    "    \"CD45RA\",\n",
+    "    \"CD133\",\n",
+    "    \"CD19\",\n",
+    "    \"CD22\",\n",
+    "    \"CD11b\",\n",
+    "    \"CD4\",\n",
+    "    \"CD8\",\n",
+    "    \"CD34\",\n",
+    "    \"Flt3\",\n",
+    "    \"CD20\",\n",
+    "    \"CXCR4\",\n",
+    "    \"CD235ab\",\n",
+    "    \"CD45\",\n",
+    "    \"CD123\",\n",
+    "    \"CD321\",\n",
+    "    \"CD14\",\n",
+    "    \"CD33\",\n",
+    "    \"CD47\",\n",
+    "    \"CD11c\",\n",
+    "    \"CD7\",\n",
+    "    \"CD15\",\n",
+    "    \"CD16\",\n",
+    "    \"CD44\",\n",
+    "    \"CD38\",\n",
+    "    \"CD13\",\n",
+    "    \"CD3\",\n",
+    "    \"CD61\",\n",
+    "    \"CD117\",\n",
+    "    \"CD49d\",\n",
+    "    \"HLA-DR\",\n",
+    "    \"CD64\",\n",
+    "    \"CD41\",\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8003a787-9ad9-4688-bd30-dda207e2507d",
+   "metadata": {},
+   "source": [
+    "We want to normalise the channels with a hyperbolic inverse sine with cofactor 5 (as performed in the original paper)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b49ccd24-5005-48cd-962d-d8c05fc6c084",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adata_arcsinh = pm.tl.normalize_arcsinh(adata[:, channels], cofactor=5, inplace=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf080288-0208-4c1e-8f2f-0206d8f36d2f",
+   "metadata": {},
+   "source": [
+    "We can now cluster our data with the `flowsom_clustering` function from the `tl` (tools) module of `pytometry`. The FlowSOM algorithm works in the following stages:\n",
+    "1. A self-organsing map is trained that learns a low dimensional embedding of the single cell feature space\n",
```

Comment:
Typo self-organising

Reply:
Please note that all notebooks in "docs/tutorials" are executed during testing. The FlowSOM notebook currently fails (due to a kernelspec issue in jupyter). I suggest that you move the notebook to "docs/examples" if you don't want to have it re-run all the time. The other option is to use a built in example dataset (Oetjen et al T cell cytof panel) provided by the `readfcs` package (cudos to Lamin Labs for providing them).